app/views/assessments/index.html.erb

```diff
@@ -1,36 +1,74 @@
-<h1>Assessments</h1>
+<div class="container">
+  <div class="panel panel-default">
+    <div class="panel-heading" style="width: 100%;"><h1 align="center">Assessments</h1></div>
+  </div>
+</div>
 
-<table>
+<table id="assessmentTable" class="display hover order-column compact" width="100%">
   <thead>
     <tr>
-      <th>Category</th>
-      <th>Name</th>
-      <th>Level</th>
-      <th>Assessment Type</th>
-      <th>Score</th>
-      <th>Assessment Date</th>
+      <th> Name </th>
+      <th> Category </th>
+      <th> Level </th>
+      <th> Assessment Type </th>
+      <th> Score </th>
+      <th> Assessment Date </th>
       <th colspan="3"></th>
     </tr>
   </thead>
 
-  <tbody>
     <% @assessments.each do |assessment| %>
       <tr>
+        <td><%= link_to assessment.name, assessment %></td>
         <td><%= assessment.category %></td>
-        <td><%= assessment.name %></td>
         <td><%= assessment.level %></td>
         <td><%= assessment.assessment_type %></td>
         <td><%= assessment.score %></td>
         <td><%= assessment.date %></td>
-        <td><%= link_to 'Show', assessment %></td>
-        <td><%= link_to 'Edit', edit_assessment_path(assessment) %></td>
-        <td><%= link_to 'Delete', assessment, method: :delete, data: { confirm: 'Are you sure?' } %></td>
+        <td class="text-right">
+          <%= link_to edit_assessment_path(assessment)do %>
+              <button class="btn btn-primary btn-sm">
+                <span class="glyphicon glyphicon-pencil"></span>
+              </button>
+          <% end %>
+          <%= link_to assessment_path(assessment), method: :delete, data: { confirm: 'Are you sure you want to delete this
+          assessment?'
+          } do %>
+              <button class="btn btn-danger btn-sm"><span class="glyphicon glyphicon-remove"></span></button>
```

Comment:
what!? icons are much more informative! they literally don't have to READ!
... oh well. if that's what she wants